### ISSUE TITLE
Dependency Dashboard

### ISSUE THREAD

**Issue by renovate[bot]** [woothemes/woocommerce#42665]
This issue lists Renovate updates and detected dependencies. Read the [Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) docs to learn more.

## Awaiting Schedule

These updates are awaiting their schedule. Click on a checkbox to get an update now.

 - [ ] <!-- unschedule-branch=renovate/lock-file-maintenance -->Lock file maintenance

## Rate Limited

These updates are currently rate limited. Click on a checkbox below to force their creation now.

 - [ ] <!-- unlimit-branch=renovate/autoprefixer-10.x -->Update dependency autoprefixer to v10.4.8
 - [ ] <!-- unlimit-branch=renovate/woocommerce-eslint-plugin-2.x -->Update dependency @woocommerce/eslint-plugin to v2.2.0
 - [ ] <!-- unlimit-branch=renovate/commander-9.x -->Update dependency commander to v9.4.0
 - [ ] <!-- unlimit-branch=renovate/eslint-import-resolver-typescript-3.x -->Update dependency eslint-import-resolver-typescript to v3.4.0
 - [ ] <!-- unlimit-branch=renovate/glob-7.x -->Update dependency glob to v7.2.3
 - [ ] <!-- unlimit-branch=renovate/mini-css-extract-plugin-1.x -->Update dependency mini-css-extract-plugin to v1.6.2
 - [ ] <!-- unlimit-branch=renovate/postcss-loader-4.x -->Update dependency postcss-loader to v4.3.0
 - [ ] <!-- unlimit-branch=renovate/puppeteer-15.x -->Update dependency puppeteer to v15.5.0
 - [ ] <!-- unlimit-branch=renovate/sass-loader-10.x -->Update dependency sass-loader to v10.3.1
 - [ ] <!-- unlimit-branch=renovate/terser-webpack-plugin-3.x -->Update dependency terser-webpack-plugin to v3.1.0
 - [ ] <!-- unlimit-branch=renovate/webpack-4.x -->Update dependency webpack to v4.46.0
 - [ ] <!-- unlimit-branch=renovate/typescript-eslint-monorepo -->Update typescript-eslint monorepo to v5.32.0 (`@typescript-eslint/eslint-plugin`, `@typescript-eslint/parser`)
 - [ ] <!-- unlimit-branch=renovate/wordpress-monorepo -->Update wordpress monorepo (`@wordpress/api-fetch`, `@wordpress/autop`, `@wordpress/babel-preset-default`, `@wordpress/base-styles`, `@wordpress/block-editor`, `@wordpress/blocks`, `@wordpress/components`, `@wordpress/compose`, `@wordpress/data-controls`, `@wordpress/dependency-extraction-webpack-plugin`, `@wordpress/deprecated`, `@wordpress/dom`, `@wordpress/e2e-test-utils`, `@wordpress/e2e-tests`, `@wordpress/element`, `@wordpress/html-entities`, `@wordpress/icons`, `@wordpress/is-shallow-equal`, `@wordpress/notices`, `@wordpress/plugins`, `@wordpress/primitives`, `@wordpress/scripts`, `@wordpress/server-side-render`, `@wordpress/url`, `@wordpress/wordcount`)
 - [ ] <!-- unlimit-branch=renovate/octokit-graphql-5.x -->Update dependency @octokit/graphql to v5
 - [ ] <!-- unlimit-branch=renovate/testing-library-react-13.x -->Update dependency @testing-library/react to v13
 - [ ] <!-- unlimit-branch=renovate/testing-library-react-hooks-8.x -->Update dependency @testing-library/react-hooks to v8
 - [ ] <!-- unlimit-branch=renovate/testing-library-user-event-14.x -->Update dependency @testing-library/user-event to v14
 - [ ] <!-- unlimit-branch=renovate/react-18.x -->Update dependency @types/react to v18
 - [ ] <!-- unlimit-branch=renovate/react-dom-18.x -->Update dependency @types/react-dom to v18
 - [ ] <!-- unlimit-branch=renovate/wordpress__block-editor-7.x -->Update dependency @types/wordpress__block-editor to v7
 - [ ] <!-- unlimit-branch=renovate/wordpress__data-6.x -->Update dependency @types/wordpress__data to v6
 - [ ] <!-- unlimit-branch=renovate/chalk-5.x -->Update dependency chalk to v5
 - [ ] <!-- unlimit-branch=renovate/copy-webpack-plugin-11.x -->Update dependency copy-webpack-plugin to v11
 - [ ] <!-- unlimit-branch=renovate/glob-8.x -->Update dependency glob to v8
 - [ ] <!-- unlimit-branch=renovate/glob-promise-5.x -->Update dependency glob-promise to v5
 - [ ] <!-- unlimit-branch=renovate/mini-css-extract-plugin-2.x -->Update dependency mini-css-extract-plugin to v2
 - [ ] <!-- unlimit-branch=renovate/postcss-loader-7.x -->Update dependency postcss-loader to v7
 - [ ] <!-- unlimit-branch=renovate/puppeteer-16.x -->Update dependency puppeteer to v16
 - [ ] <!-- unlimit-branch=renovate/sass-loader-13.x -->Update dependency sass-loader to v13
 - [ ] <!-- unlimit-branch=renovate/terser-webpack-plugin-5.x -->Update dependency terser-webpack-plugin to v5
 - [ ] <!-- unlimit-branch=renovate/webpack-5.x -->Update dependency webpack to v5
 - [ ] <!-- unlimit-branch=renovate/webpack-cli-4.x -->Update dependency webpack-cli to v4
 - [ ] <!-- unlimit-branch=renovate/major-jest-monorepo -->Update jest monorepo to v28 (major) (`@types/jest`, `jest-circus`)
 - [ ] <!-- unlimit-branch=renovate/major-wordpress-monorepo -->Update wordpress monorepo (major) (`@wordpress/block-editor`, `@wordpress/env`, `@wordpress/icons`, `@wordpress/scripts`)
 - [ ] <!-- unlimit-branch=renovate/actions-setup-node-digest -->Update actions/setup-node digest to 2a814b5
 - [ ] <!-- unlimit-branch=renovate/actions-upload-artifact-digest -->Update actions/upload-artifact digest to 3cea537
 - [ ] <!-- unlimit-branch=renovate/node-16.x -->Update Node.js to v16.16.0
 - [ ] <!-- unlimit-branch=renovate/mariadb-10.x -->Update dependency mariadb to v10.8
 - [ ] <!-- unlimit-branch=renovate/actions-cache-3.x -->Update actions/cache action to v3
 - [ ] <!-- unlimit-branch=renovate/html-react-parser-3.x -->Update dependency html-react-parser to v3
 - [ ] <!-- unlimit-branch=renovate/use-debounce-8.x -->Update dependency use-debounce to v8
 - [ ] <!-- unlimit-branch=renovate/wordpress-components-19.x -->Update dependency wordpress-components to v19
 - [ ] <!-- unlimit-branch=renovate/major-react-monorepo -->Update react monorepo to v18 (major) (`react`, `react-dom`, `react-test-renderer`)

## Ignored or Blocked

These are blocked by an existing closed PR and will not be recreated unless you click a checkbox below.

 - [ ] <!-- recreate-branch=renovate/npm-node-fetch-vulnerability -->[Update dependency node-fetch to 2.6.7 [SECURITY]](../pull/5805)
 - [ ] <!-- recreate-branch=renovate/npm-nth-check-vulnerability -->[Update dependency nth-check to 2.0.1 [SECURITY]](../pull/5085)
 - [ ] <!-- recreate-branch=renovate/pin-dependencies -->[Pin dependencies](../pull/5921) (`babel-plugin-explicit-exports-references`, `eslint`)
 - [ ] <!-- recreate-branch=renovate/babel-monorepo -->[Update babel monorepo to v7.18.10](../pull/6820) (`@babel/cli`, `@babel/core`)
 - [ ] <!-- recreate-branch=renovate/react-17.x -->[Update dependency @types/react to v17.0.48](../pull/6821)
 - [ ] <!-- recreate-branch=renovate/core-js-3.x -->[Update dependency core-js to v3.24.1](../pull/6720)

## Detected dependencies

<details><summary>docker-compose</summary>
<blockquote>

<details><summary>docker-compose.yml</summary>

 - `mariadb 10.5`

</details>

</blockquote>
</details>

<details><summary>dockerfile</summary>
<blockquote>

<details><summary>bin/docker/wp-cli/Dockerfile</summary>

 - `wordpress cli-php7.4`

</details>

</blockquote>
</details>

<details><summary>github-actions</summary>
<blockquote>

<details><summary>.github/workflows/approved-with-labels.yml</summary>

 - `pullreminders/label-when-approved-action master`
 - `actions/checkout v3`
 - `woocommerce/automations v1`

</details>

<details><summary>.github/workflows/automate-team-review-assignment-config.yml</summary>

 - `acq688/Request-Reviewer-For-Team-Action v1.1`

</details>

<details><summary>.github/workflows/bundle-size.yml</summary>

 - `actions/checkout v3`
 - `actions/setup-node v3`
 - `preactjs/compressed-size-action v2`

</details>

<details><summary>.github/workflows/check-modified-assets.yml</summary>

 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `actions/upload-artifact v3`
 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `actions/download-artifact v3`

</details>

<details><summary>.github/workflows/close-stale-issues.yml</summary>

 - `actions/stale v5`

</details>

<details><summary>.github/workflows/codeql-analysis.yml</summary>

 - `actions/checkout v3`
 - `github/codeql-action v2`
 - `github/codeql-action v2`
 - `github/codeql-action v2`

</details>

<details><summary>.github/workflows/flaky-tests.yml</summary>

 - `actions/checkout v3`
 - `actions/setup-node 5b52f097d36d4b0b2f94ed6de710023fbb8b2236`

</details>

<details><summary>.github/workflows/generate-zip.yml</summary>

 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `AbleLincoln/push-to-sftp v1.2`
 - `mshick/add-pr-comment v1`

</details>

<details><summary>.github/workflows/js-css-md-linting.yml</summary>

 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `actions/upload-artifact v3`
 - `ataylorme/eslint-annotate-action 1.2.0`
 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `actions/checkout v3`
 - `actions/setup-node v3`

</details>

<details><summary>.github/workflows/php-coding-standards.yml</summary>

 - `actions/checkout v3`
 - `actions/cache v2`
 - `shivammathur/setup-php v2`

</details>

<details><summary>.github/workflows/php-js-e2e-tests.yml</summary>

 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `actions/cache v2`
 - `shivammathur/setup-php v2`
 - `actions/upload-artifact v3`
 - `actions/upload-artifact e448a9b857ee2131e752b06002bf0e093c65e571`
 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `actions/cache v2`
 - `shivammathur/setup-php v2`
 - `actions/upload-artifact v3`
 - `actions/upload-artifact e448a9b857ee2131e752b06002bf0e093c65e571`

</details>

<details><summary>.github/workflows/project-management-automations.yml</summary>

 - `actions/checkout v3`
 - `woocommerce/automations v1`

</details>

<details><summary>.github/workflows/release-pull-request.yml</summary>

 - `act10ns/slack v1`
 - `actions/checkout v3`
 - `woocommerce/automations v1`
 - `act10ns/slack v1`

</details>

<details><summary>.github/workflows/unit-tests.yml</summary>

 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `actions/upload-artifact v3`
 - `actions/cache v2`
 - `shivammathur/setup-php v2`
 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `actions/download-artifact v3`
 - `actions/cache v2`
 - `actions/checkout v3`
 - `actions/cache v3`
 - `actions/setup-node v3`
 - `actions/download-artifact v3`

</details>

<details><summary>.github/workflows/wordpress-deploy.yml</summary>

 - `act10ns/slack v1`
 - `actions/checkout v3`
 - `10up/action-wordpress-plugin-deploy stable`
 - `actions/upload-release-asset v1`
 - `act10ns/slack v1`

</details>

</blockquote>
</details>

<details><summary>npm</summary>
<blockquote>

<details><summary>bin/eslint-plugin-woocommerce/package.json</summary>

 - `eslint ^8.3.0`
 - `node ^16.13.0`
 - `npm ^8.0.0`

</details>

<details><summary>package.json</summary>

 - `@wordpress/autop 3.12.0`
 - `@wordpress/compose 5.5.0`
 - `@wordpress/deprecated 3.12.0`
 - `@wordpress/icons 6.1.1`
 - `@wordpress/notices 3.12.0`
 - `@wordpress/plugins 4.10.0`
 - `@wordpress/primitives 3.0.4`
 - `@wordpress/server-side-render 3.10.0`
 - `@wordpress/url 3.13.0`
 - `@wordpress/wordcount 3.12.0`
 - `classnames 2.3.1`
 - `compare-versions 4.1.3`
 - `config 3.3.7`
 - `dataloader 2.1.0`
 - `dinero.js 1.9.1`
 - `downshift 6.1.7`
 - `html-react-parser 0.14.3`
 - `react-number-format 4.9.3`
 - `reakit 1.3.11`
 - `snakecase-keys 5.4.2`
 - `trim-html 0.1.9`
 - `use-debounce 7.0.1`
 - `wordpress-components 14.2.0`
 - `@actions/core 1.9.0`
 - `@actions/github 5.0.3`
 - `@automattic/color-studio 2.5.0`
 - `@babel/cli 7.18.9`
 - `@babel/core 7.18.9`
 - `@babel/plugin-proposal-class-properties 7.18.6`
 - `@babel/plugin-syntax-jsx 7.18.6`
 - `@babel/polyfill 7.12.1`
 - `@babel/preset-typescript 7.18.6`
 - `@octokit/graphql 4.8.0`
 - `@storybook/addon-a11y 6.5.9`
 - `@storybook/addon-essentials 6.5.9`
 - `@storybook/addon-links 6.5.9`
 - `@storybook/addon-storysource 6.5.9`
 - `@storybook/addons 6.5.9`
 - `@storybook/client-api 6.5.9`
 - `@storybook/react 6.5.9`
 - `@testing-library/jest-dom 5.16.4`
 - `@testing-library/react 12.1.5`
 - `@testing-library/react-hooks 7.0.2`
 - `@testing-library/user-event 13.5.0`
 - `@types/classnames 2.3.0`
 - `@types/dinero.js 1.9.0`
 - `@types/gtag.js 0.0.10`
 - `@types/jest 27.5.2`
 - `@types/jest-environment-puppeteer 5.0.2`
 - `@types/jquery 3.5.14`
 - `@types/lodash 4.14.182`
 - `@types/puppeteer 5.4.6`
 - `@types/react 17.0.47`
 - `@types/react-dom 17.0.17`
 - `@types/wordpress__block-editor 6.0.6`
 - `@types/wordpress__blocks 11.0.5`
 - `@types/wordpress__compose 4.0.1`
 - `@types/wordpress__data 4.6.11`
 - `@types/wordpress__data-controls 2.2.0`
 - `@typescript-eslint/eslint-plugin 5.30.5`
 - `@typescript-eslint/parser 5.30.5`
 - `@woocommerce/api 0.2.0`
 - `@woocommerce/e2e-utils 0.2.0`
 - `@woocommerce/eslint-plugin 2.0.0`
 - `@woocommerce/woocommerce-rest-api 1.0.1`
 - `@wordpress/api-fetch 6.9.0`
 - `@wordpress/babel-preset-default 6.14.0`
 - `@wordpress/base-styles 4.0.4`
 - `@wordpress/block-editor 8.2.0`
 - `@wordpress/blocks 11.1.5`
 - `@wordpress/browserslist-config 4.1.2`
 - `@wordpress/components 19.1.5`
 - `@wordpress/data-controls 2.2.7`
 - `@wordpress/dependency-extraction-webpack-plugin 3.2.1`
 - `@wordpress/dom 3.2.7`
 - `@wordpress/e2e-test-utils 7.8.0`
 - `@wordpress/e2e-tests 4.6.0`
 - `@wordpress/element 4.0.4`
 - `@wordpress/env 4.9.0`
 - `@wordpress/html-entities 3.12.0`
 - `@wordpress/i18n 4.14.0`
 - `@wordpress/is-shallow-equal 4.13.0`
 - `@wordpress/prettier-config 1.4.0`
 - `@wordpress/scripts 22.3.0`
 - `autoprefixer 10.4.7`
 - `axios 0.27.2`
 - `babel-plugin-explicit-exports-references ^1.0.2`
 - `babel-plugin-react-docgen 4.2.1`
 - `babel-plugin-transform-async-generator-functions 6.24.1`
 - `babel-plugin-transform-object-rest-spread 6.26.0`
 - `babel-plugin-transform-react-jsx 6.24.1`
 - `babel-plugin-transform-react-remove-prop-types 0.4.24`
 - `chalk 4.1.2`
 - `circular-dependency-plugin 5.2.2`
 - `commander 9.3.0`
 - `copy-webpack-plugin 6.4.1`
 - `core-js 3.23.3`
 - `create-file-webpack 1.0.2`
 - `cross-env 7.0.3`
 - `cssnano 5.1.12`
 - `deep-freeze 0.0.1`
 - `eslint-import-resolver-typescript 3.2.4`
 - `eslint-import-resolver-webpack 0.13.2`
 - `eslint-plugin-import 2.26.0`
 - `eslint-plugin-woocommerce file:bin/eslint-plugin-woocommerce`
 - `eslint-plugin-you-dont-need-lodash-underscore 6.12.0`
 - `expect-puppeteer 6.1.1`
 - `follow-redirects 1.15.1`
 - `fs-extra 10.1.0`
 - `gh-pages 4.0.0`
 - `github-label-sync 2.2.0`
 - `glob 7.1.7`
 - `glob-promise 4.2.2`
 - `husky 2.4.1`
 - `ignore-loader 0.1.2`
 - `jest-circus 27.5.1`
 - `jest-environment-puppeteer 6.1.1`
 - `jest-fetch-mock 3.0.3`
 - `jest-html-reporters 3.0.10`
 - `json2md 1.12.0`
 - `lint-staged 13.0.3`
 - `lodash 4.17.21`
 - `markdown-it 13.0.1`
 - `merge-config 2.0.0`
 - `mini-css-extract-plugin 1.3.6`
 - `patch-package 6.4.7`
 - `postcss 8.4.14`
 - `postcss-loader 4.2.0`
 - `prettier 2.6.2`
 - `progress-bar-webpack-plugin 2.1.0`
 - `promptly 3.2.0`
 - `puppeteer 15.3.2`
 - `react-docgen 5.4.3`
 - `react-test-renderer 17.0.2`
 - `request-promise 4.2.6`
 - `rimraf 3.0.2`
 - `sass-loader 10.1.0`
 - `storybook-addon-react-docgen 1.2.42`
 - `terser-webpack-plugin 3.0.3`
 - `typescript 4.7.4`
 - `utility-types 3.10.0`
 - `webpack 4.44.2`
 - `webpack-bundle-analyzer 4.5.0`
 - `webpack-cli 3.3.12`
 - `webpack-rtl-plugin 2.0.0`
 - `wp-types 3.60.0`
 - `zenhub-api 0.2.0`
 - `ndb 1.1.5`
 - `react ^17.0.0`
 - `react-dom ^17.0.0`
 - `node ^16.13.0`
 - `npm ^8.0.0`

</details>

</blockquote>
</details>

<details><summary>nvm</summary>
<blockquote>

<details><summary>.nvmrc</summary>

 - `node 16.13.2`

</details>

</blockquote>
</details>

---

- [ ] <!-- manual job -->Check this box to trigger a request for Renovate to run again on this repository

**Comment by timmy5685** [comment]
Closing this out — it's an auto-generated Renovate dependency dashboard from the pre-monorepo era, and Renovate is no longer active on this repo. The dashboard hasn't been refreshed since ~2021–2022 (it still lists webpack v4, React v17, node 16, and the since-removed close-stale-issues.yml workflow), and the renovate.json still references the separate repos — woocommerce-admin, woocommerce-blocks, and friends — that were long ago merged into this monorepo.

Closing as obsolete. The leftover renovate.json could be cleaned up separately.
